### DISCUSSION TITLE
Continuous Increase/Decrease Issue with Number Input in Custom Edge

### DISCUSSION THREAD

**Discussion by kohjean** [xyflow/xyflow#3174] [General]
I am currently working on creating a custom edge using the `<EdgeLabelRenderer />` component. Within this custom edge, I have implemented a number type input element with an `onChange` handler that updates the state with the value from the event.

The problem arises when I interact with the up/down arrows on the number input. Upon clicking these arrows, the value continuously increases/decreases without stopping.

However, this behavior does not occur when using the keyboard up and down keys to trigger the `onChange` event.

It seems that this continuous increment/decrement behavior persists until the input element loses focus.

I have created an open sandbox that demonstrates this issue:
[Link to sandbox](https://codesandbox.io/s/prod-dust-v48vcr)

I would greatly appreciate any ideas or suggestions on how to resolve this issue.

Environment:
- OS: Windows 10 Pro 22/H2
- Browser: 
  - Chrome: 114.0.5735.134（Official Build）
  - Edge: 114.0.1823.58 (Official Build)

Steps to reproduce:
1. Go to the provided sandbox link.
2. Interact with the custom edge's number input using the up/down arrows.
3. Observe the continuous increment/decrement behavior.

Expected behavior:
The value should only increment/decrement once when using the up/down arrows on the number input, similar to the behavior when using the keyboard up and down keys.

Thank you for your attention to this issue.

**Comment by moklick** [comment]
Hey @kohjean 

you need to add the class name "nopan" https://codesandbox.io/s/optimistic-darwin-3xdzzt?file=/CustomEdge.tsx:590-618

**Reply by kohjean** [reply]
Big up!! Thank you @moklick

**Reply by Gilbishkosma** [reply]
Thanks, i got the same issue today, got fixed with this.

**Reply by Gilbishkosma** [reply]
I added the "nodrag" class
